backend/src/main/java/gov/cdc/usds/simplereport/api/FlexibleDateScalarType.java

```diff
@@ -15,10 +16,18 @@
   private static final DateTimeFormatter US_DASHDATE_FORMATTER = DateTimeFormatter.ISO_LOCAL_DATE;
   private static final DateTimeFormatter US_SLASHDATE_FORMATTER =
       DateTimeFormatter.ofPattern("M/d/yyyy");
+  private static final DateTimeFormatter US_SLASHDATE_TWO_DIGIT_YEAR_FORMATTER =
+      DateTimeFormatter.ofPattern("M/d/yy");
+  private static final int CENTURY = 100;
 
   LocalDate convertImpl(Object input) {
     if (input instanceof String) {
       if (((String) input).contains("/")) {
```

Comment:
**nitpick (non-blocking)**: Since this `(String) input` cast is performed so many times, we might DRY it out with a `var inputStr = (String) input` right below the initial `instanceof` check. That code was already here, so I'm good w/ it as-is though